lib/gs1/content.rb

```diff
@@ -0,0 +1,47 @@
+module GS1
+  # Identification of trade items contained in a logistic unit: AI (02)
+  #
+  # The GS1 Application Identifier (02) indicates that the GS1 Application Identifier data field includes
+  # the GTIN of the contained trade items. The GTIN is used to identify trade items (see section 4).
+  #
+  # The GTIN for trade items may be a GTIN-8, GTIN-12, GTIN-13 or a GTIN-14. See section 2 for the
+  # rules for GTIN formats and mandatory or optional attributes in the various trade item applications.
+  #
+  # The GTIN of the trade items contained is the GTIN of the highest level of trade item contained in the
+  # logistic unit.
+  #
+  # Note: This element string SHALL be used only on a logistic unit if:
+  #
+  # - the logistic unit is not itself a trade item; and
+  # - all trade items that are contained at the highest level have the same GTIN
+  #
+  # The check digit is explained in section 7.9. Its verification, which must be carried out in the
```

Comment:
Where is this section?

Reply:
In https://www.gs1.org/sites/default/files/docs/barcodes/GS1_General_Specifications.pdf as referenced in the bottom of the description.

Reply:
please review the newest changes